tests/karma.js

```diff
@@ -556,6 +556,108 @@ beforeAll(function (done) {
             '</ocs>\n'
         }
       }))
+    .then(() =>
+      provider.addInteraction({
+        uponReceiving: 'apps GET request',
```

Comment:
please rename, to make clear its a request with an invalid auth

Reply:
fixed